srcpkgs/sqlitestudio/files/sqlitestudio.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=SQLiteStudio
+Comment=Database manager to create, edit, and browse SQLite databases
+Icon=/usr/share/icons/hicolor/scalable/apps/sqlitestudio.svg
```

```suggestion
Icon=sqlitestudio
```
I think this could work